scripts/qa_crowbarsetup.sh

```diff
@@ -3491,6 +3495,15 @@ function oncontroller_run_tempest()
     fi
     testr last --subunit | subunit-1to2 > tempest.subunit.log
 
+    if [ -n "$ostestroptions" ]; then
+        zypper -n in python-os-testr
+        ostestr $ostestroptions 2>&1 | tee ostestr.log
```

Comment:
I am thinking that maybe we want to integrate it into Rally, you can see that tempest is stored in subunit format and send to the backup for QA, this is the Rally server.